constraint/pkg/client/drivers/local/driver_unit_test.go

```diff
@@ -171,18 +181,179 @@ func TestDriver_Query(t *testing.T) {
 	if len(res) == 0 {
 		t.Fatalf("got 0 errors on data-less query; want 1")
 	}
+}
 
-	stats, ok := res[0].EvaluationMeta.(RegoEvaluationMeta)
-	if !ok {
-		t.Fatalf("could not type convert to RegoEvaluationMeta")
+// TestDriver_Query_StatsEntries tests that StatsEntries are populated for Query calls.
```

Comment:
I went w a longer test, split test here for readability. IMO, there are not that many cases to test here and a table test may be confusing. If we are adding more and more cases, then I can be persuaded to convert these.